test/Google.Datastore.V1Beta3.IntegrationTests/DatastoreDbTest.cs

```diff
@@ -85,5 +87,18 @@ public void Lookup_NoPartition()
                 db.Delete(insertedKey);
             }
         }
+
+        [Fact]
+        public void RunQuery_NoResults()
+        {
+            var db = DatastoreDb.Create(_fixture.ProjectId, _fixture.NamespaceId);
+            var query = db.RunQuery(new Query("absent"));
+            Assert.Equal(0, query.Count());
```

Comment:
This is running the query multiple times?
If so, might be worth a comment, just in case anyone looks at this code.